ui/src/message_popup/mobile_prototype/mobile_interface.jsx

```diff
@@ -44,29 +44,29 @@ export default React.createClass({
     const goodExamples = _.map(this.props.question.examples, (example) => { return {type: 'Good', text: example}; });
     const badExamples = _.map(this.props.question.nonExamples, (example) => { return {type: 'Bad', text: example}; });
     const allExamples = _.shuffle(_.clone(goodExamples).concat(badExamples)); 
-    const messages = this.getInitialMessages(this.props.question);
     return ({
-      messages,
+      messages: [],
       allExamples,
       initialResponse: undefined,
       revisedResponse: undefined,
       elapsedMs: 0,
       exampleIndex: 0,
       dialog: null,
-      messageIndex: 1
+      messageIndex: 1,
+      disabledButton: false
```

Comment:
disabledButton was added so that animations could finish before sending in another message. Without this, users could beat the revision/example message with another message leading to two user messages before the revision/example message.